config/autoload/cors.global.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+use Mezzio\Cors\Configuration\ConfigurationInterface;
+
+return [
+    ConfigurationInterface::CONFIGURATION_IDENTIFIER => [
+        'allowed_origins' => [ConfigurationInterface::ANY_ORIGIN], // Allow any origin
```

Comment:
In the initial implementation, the configuration was stored in local to prevent `*` from being used in production unless it's intended.

However this is fine if the consuming frontend is on a different domain. (eg.: myfrontend.com calls -> api.dotkernel.com )
shouldn't `allowed_origins` key be added in the local config?

'allowed_origins' => [ConfigurationInterface::ANY_ORIGIN], // Allow any origin

https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS

https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS/Errors/CORSMissingAllowOrigin

Reply:
Moved `allowed_origins` config to local.php